samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt

```diff
@@ -42,17 +42,12 @@ class EmojiSearchActivity : ComponentActivity() {
     super.onCreate(savedInstanceState)
 
     val treehouseApp = createTreehouseApp()
+    val treehouseContent = TreehouseView.Content(EmojiSearchPresenter::launch)
```

Comment:
I could be wrong, but I think `TreehouseView.Content` might always be considered unstable since the interface is defined in a module that doesn't apply the compose compiler. I enabled compose metrics and the `TreehouseContent` definition says it's unstable:

```
restartable scheme("[androidx.compose.ui.UiComposable]") fun TreehouseContent(
  unstable treehouseApp: TreehouseApp<T>
  unstable content: Content<T>
)
```

Reply:
Well I guess that's something to fix in the future